src/styles/getMuiTheme.js

```diff
@@ -230,7 +232,9 @@ export default function getMuiTheme(baseTheme, muiTheme) {
     },
   }, muiTheme);
 
+  const transformers = [autoprefixer, rtl, callOnce].filter(t => t(muiTheme));
```

Comment:
you're initializing the transformers twice. it can have unintended side-effects.

```js
[autoprefixer, rtl, callOnce].map(t => t(muiTheme)).filter(t => t);
// and then
compose(...transformers);
```

Would be safer